modules/vnet/main.tf

```diff
@@ -28,14 +28,14 @@ resource "azurerm_subnet" "this" {
   address_prefixes     = each.value.address_prefixes
 }
 
-resource "azurerm_network_security_group" "this" {
-  for_each = var.network_security_groups
+# resource "azurerm_network_security_group" "this" {
+#   for_each = var.network_security_groups
 
-  name                = each.key
-  location            = try(each.value.location, var.location)
-  resource_group_name = var.resource_group_name
-  tags                = var.tags
-}
+#   name                = each.key
+#   location            = try(each.value.location, var.location)
+#   resource_group_name = var.resource_group_name
+#   tags                = var.tags
+# }
 
 locals {
```

Comment:
This local value is no longer used, please remove it.

Reply:
fixed